src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -233,7 +233,6 @@ initialize_query_table()
   emplace_sysfs_get<query::xclbin_uuid>                 ("", "xclbinuuid");
   emplace_sysfs_get<query::mem_topology_raw>            ("icap", "mem_topology");
   emplace_sysfs_get<query::ip_layout_raw>               ("icap", "ip_layout");
-  emplace_sysfs_get<query::clock_freq_topology_raw>     ("icap", "clock_freq_topology");
```

Comment:
@AShivangi Why was this removed?

Reply:
I must have removed it when I was resolving merge conflicts. You can add it back.